pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/FloatingPointDataTypeTest.java

```diff
@@ -39,7 +39,7 @@
 /**
  * Integration test for floating point data type (float & double) filter queries.
  */
-@Test(suiteName = "CustomClusterIntegrationTest")
+@Test(suiteName = "CustomClusterIntegrationTest", groups = {"custom-integration-suite"})
```

Comment:
so before this change the @BeforeSuite and @BeforeClass is basically the same b/c they are not group into suite is that correct? (we should see huge test time reduction if so)

Reply:
Only this package has the suite implementation, which is already tested using `mvn test -P github-actions,custom-cluster-integration-test-suite || exit 1`. But in this case, we are missing the code coverage report for this.
So just wanna merge these two suite to run together.